components/verify/Verifier.tsx

```diff
@@ -66,13 +66,29 @@ const Verifier: React.FC<VerifierProps> = ({ wrappedDocument }) => {
 
       setFragmentVerificationStatus({ DOCUMENT_STATUS, DOCUMENT_INTEGRITY, ISSUER_IDENTITY });
 
+      const docStatusFragments = fragments.filter(
+        (fragment) => fragment.type === "DOCUMENT_STATUS" && fragment.status === "INVALID"
+      ) as InvalidVerificationFragment<{}>[];
+      const isRevoked = docStatusFragments.some((fragment) => fragment.reason.codeString === "DOCUMENT_REVOKED");
+
       if (isHealthCert(document)) {
-        setCustomMessage({
+        let customMessage = {};
+        if (isRevoked) {
```

Comment:
Do we only want to display this message for healthcerts? if not I can extract it outside of the `(isHealthCert clause)`

Reply:
I had the same thought too. Shall we clarify with Darren?

Reply:
changed to reflect for all documents as stated within the story